front/app/containers/CustomPageShow/index.tsx

```diff
@@ -36,19 +36,40 @@ const PageTitle = styled.h1`
   text-align: left;
   margin: 0;
   padding: 0;
-  padding-top: 50px;
 
   ${media.tablet`
-    font-size: ${fontSizes.xxxl};
+    font-size: ${fontSizes.xxxl}px;
   `}
+
   ${isRtl`
     text-align: right;
     direction: rtl;
   `}
 `;
 
-const AttachmentsContainer = styled(ContentContainer)`
-  margin-bottom: 30px;
+const AttachmentsContainer = styled(ContentContainer)<{
+  topInfoSectionEnabled: boolean;
+}>`
+  background: #fff;
+  padding-top: ${({ topInfoSectionEnabled }) =>
+    topInfoSectionEnabled ? '0' : '50px'};
```

Comment:
So, currently, we add paddings/margins both on top and bottom (left and right)? It seems previously, the rule was to add it only on the bottom and right, wasn't it?

Reply:
That's the case for components (but it's not 100% strict either). It's just to avoid collapsing margins. However, this is a page layout with sections, a different beast. It already had the top/right/bottom/left. I just changed it.